nil/internal/types/signature.go

```diff
@@ -33,3 +34,16 @@ func (s Signature) Hex() string {
 func (s Signature) String() string {
 	return s.Hex()
 }
+
+type BlsSignature []byte
+
+type BlsAggregateSignature struct {
+	Sig  []byte `json:"sig" yaml:"sig" ssz-max:"64"`
+	Mask []byte `json:"mask" yaml:"mask" ssz-max:"128"`
```

Comment:
By default `json` serializes it like an array `[1 2 3]`. Please sure it's so and implement MarshalText, UnmarshalText methods... Or you can use hexutil.Bytes instead of []byte